app/views/metrics/show.html.erb

```diff
@@ -21,12 +21,14 @@
                   {
                     header: t(".navigation.visit_page"),
                     link_url: "//www.gov.uk#{@performance_data.metadata[:base_path]}",
-                    link_text: t('.navigation.visit_link', content_title: @performance_data.title)
+                    accessibility_message: t('.navigation.accessibility_message'),
+                    label: @performance_data.title
                   },
                   {
                     header: "Make changes to the page",
                     link_url: "#",
-                    link_text: @performance_data.publishing_app
+                    accessibility_message: "",
+                    label: @performance_data.publishing_app
```

Comment:
Should we use the locales for the edit link lable? In the design it shows with `Edit <publisher app>`